lifecycle-operator/apis/lifecycle/v1alpha3/keptntaskdefinition_types.go

```diff
@@ -142,3 +154,17 @@ type KeptnTaskDefinitionList struct {
 func init() {
 	SchemeBuilder.Register(&KeptnTaskDefinition{}, &KeptnTaskDefinitionList{})
 }
+
+func (d *KeptnTaskDefinition) GetServiceAccount() string {
+	if d.Spec.ServiceAccount == nil {
+		return ""
+	}
+	return d.Spec.ServiceAccount.Name
+}
+
+func (d *KeptnTaskDefinition) GetAutomountServiceAccountToken() *bool {
+	if d.Spec.AutomountServiceAccountToken == nil {
+		return nil
+	}
+	return d.Spec.AutomountServiceAccountToken.Type
+}
```

Comment:
Please introduce unit tests also for these two functions

Reply:
hey @odubajDT, where to put the test for these functions ?